symfony/plugins/orangehrmCorePlugin/test/Dao/MenuDaoTest.php

```diff
@@ -187,4 +187,14 @@ public function testEnableModuleMenuItemsByTitle()
         $count = $statement->fetchOne();
         $this->assertEquals(0, $count);
     }
+
+    public function testGetMenuLevel()
```

Comment:
```php
public function testGetMenuLevel(): void
```

Reply:
done, on that file, :void not added for the function, so i followed that way.